### PR TITLE
Fix outdated use of the sf::VideoMode constructor

### PR DESCRIPTION
**Pull request by ChristianIvicevic**
## Description

Fixes the use of the `sf::VideoMode` constructor in the documentation that no longer accepts separate dimensions and instead requires a vector.

### CODE REVIEW

**Review by ChrisThrasher — APPROVED** [review]
Thanks! We can merge this once another maintainer gets a chance to look at this.

**Comment by ChrisThrasher** [comment]
SFMLBuildMaster: Build this please.

**Review by binary1248 — APPROVED** [review]

**Comment by ChrisThrasher** [comment]
Turns out there are many more places where this didn't get updated. I'm using this regex to find other instances to fix: `VideoMode\([0-9]`. This turned up 13 results in 11 files. Can we fix all of those in this one PR? Sorry to expand the scope but it'll be great to fix this problem outright.

**Comment by ChristianIvicevic** [comment]
@ChrisThrasher One of the occurences was a false positive, but I have changed the rest of it which was mostly templates or docs.

**Comment by eXpl0it3r** [comment]
SFMLBuildMaster: Build this please.

**Review by eXpl0it3r — APPROVED** [review]

**Comment by eXpl0it3r** [comment]
Thanks for this clean-up! 🙂

**Pull request merged**
